inlang/source-code/ide-extension/src/services/error-monitoring/implementation.ts

```diff
@@ -0,0 +1,15 @@
+import * as Sentry from "@sentry/node"
+import { version } from "../../../package.json"
+import { isProduction } from "./env.js"
+
+export function initErrorMonitoring() {
+	Sentry.init({
+		dsn: "https://paste-sentry-url.here",
```

Comment:
replace sentry url here after creating the project in sentry
cc @samuelstroschein